apps/api/src/project/service/project.service.ts

```diff
@@ -335,13 +337,27 @@ export class ProjectService {
     projectId: Project['id'],
     forkMetadata: ForkProject
   ) {
-    const project =
-      await this.authorityCheckerService.checkAuthorityOverProject({
-        userId: user.id,
-        entity: { id: projectId },
-        authority: Authority.READ_PROJECT,
-        prisma: this.prisma
-      })
+    const project = await this.prisma.project.findUnique({
+      where: { id: projectId },
+      select: {
+        id: true,
+        name: true,
+        description: true,
+        storePrivateKey: true,
+        accessLevel: true,
+        privateKey: true
+      }
+    })
+
+    if (!project) {
+      throw new NotFoundException(`Project with id ${projectId} not found`)
+    }
+
+    if (project.accessLevel !== 'GLOBAL') {
```

Comment:
Instead of the string, use `ProjectAccessLevel.Global`

Reply:
done